advanced/list-of-log-names.md

```diff
@@ -1,72 +1,76 @@
 ---
-description: In this page all the available logs are listed
+description: On this page all the available logs are listed
 ---
 
 # List of log names
 
-## Categories
+Logs, categorized by what they do are listed here, and are **case sensitive**. Individual logs are written in _UPPER\_CASE_, while category names(which can be used in place of individual log names, see [configuring channels](../admin-commands/config/configuring-channels.md)) are _Capitalized_. Each one of them is separated by categories in the lists below. If you want to log everything, use an asterisk to include all things that can be logged, including ones that will be introduced in future updates.
 
 {% hint style="warning" %}
-Log names are **case sensitive**. Individual logs are written in _UPPER\_CASE_, while category names are _Capitalized_
+Some logs require the bot to have the "View Audit Logs" permission enabled.
 {% endhint %}
 
-{% hint style="info" %}
-If you want everything to be logged, use `*` as name. This will also include logs that will be introduced **after** you set it.
-{% endhint %}
+## Moderation
+
+This category is used for actions by moderators or ones that moderators could use. Most of them are fired when a command is run unless they start with AUDIT_. In that case, actions that are done with or without using the bot are logged.
+
+| Name              | Emitted when     |
+| ----------------- | ---------------- |
+| BAN               | Someone gets banned |
```

Comment:
"someone" in much of these cases is too generic, try using more accurate terms like "A user". Use passive voice too